src/main/java/org/opentripplanner/transit/service/TransitModel.java

```diff
@@ -556,6 +560,53 @@ public FlexTrip getFlexTrip(FeedScopedId tripId) {
     return flexTripsById.get(tripId);
   }
 
+  public Set<StopLocation> getStopLocationsUsedForFlexTrips() {
```

Comment:
@t2gran needs to say if this is allowed here or if it's too use-case specific.

Reply:
Since both of these methods, getStopLocationsUsedForFlexTrips() and getStopLocationsUsedForCarsAllowedTrips() are pretty specific and not generally usable I would keep them in the StreetLinkerModule unless there's a good reason not to.

Reply:
I would have a reason to use `getStopLocationsUsedForCarsAllowedTrips()` in the follow-up pr related to filtering car transfers from the `TransitModel`. I would not have a use for `getStopLocationsUsedForFlexTrips()` in that pr. Would it be ok to add `getStopLocationsUsedForCarsAllowedTrips()` to the `TransitModel`, but leave `getStopLocationsUsedForFlexTrips()` in the `StreetLinkerModule`?

Reply:
That would be fine by me.

Reply:
That sounds fine to me too

Reply:
This is now changed to the suggestion I wrote, `getStopLocationsUsedForFlexTrips()` is in the StreetLinkerModule and `getStopLocationsUsedForCarsAllowedTrips()` is in the TransitModel